### ISSUE TITLE
[Bug] - Can't run npm commands from Installation page with Node 7.0.0

### ISSUE THREAD

**Issue by matthamil** [adonisjs/adonis-framework#358]
Node: v7.0.0
npm: 3.10.8

I'm trying to follow along on the [Installation](http://www.adonisjs.com/docs/3.1/installation) page, and I'm getting errors trying to run the app.

I was able to run `adonis new yardstick` and build the new app, but `npm run dev` throws the following error in my terminal:

```bash
> adonis-app@3.1.0 dev /Users/matthamil/Programming/Adonis/yardstick
> nodemon --watch app --watch bootstrap --watch config --watch .env -x "node --harmony_proxies" server.js

[nodemon] 1.11.0
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: /Users/matthamil/Programming/Adonis/yardstick/app/**/* /Users/matthamil/Programming/Adonis/yardstick/bootstrap/**/* /Users/matthamil/Programming/Adonis/yardstick/config/**/* .env
[nodemon] starting `node --harmony_proxies server.js`
node: bad option: --harmony_proxies
[nodemon] app crashed - waiting for file changes before starting...
```

I'm not sure if the `--harmony_proxies` flag was removed from Node v7+. I am able to run `node server.js` and the scaffolded app works fine.

## Solution for those running Node v7+

I changed my npm scripts to the following and now they work fine:

```json
{
  "name": "adonis-app",
  "version": "3.1.0",
  "main": "index.js",
  "scripts": {
    "dev": "nodemon --watch app --watch bootstrap --watch config --watch .env -x \"node \" server.js",
    "start": "node server.js",
    "lint": "standard"
  },
  "author": "",
  "license": "MIT",
  "description": "",
  "private": true,
  "dependencies": {
    "adonis-ace": "^3.0.4",
    "adonis-auth": "^1.0.4",
    "adonis-commands": "^2.1.2",
    "adonis-fold": "^3.0.3",
    "adonis-framework": "^3.0.4",
    "adonis-lucid": "^3.0.10",
    "adonis-middleware": "^1.0.8",
    "standard": "^8.4.0",
    "youch": "^1.0.1"
  },
  "standard": {
    "global": [
      "use",
      "make"
    ]
  },
  "autoload": {
    "App": "./app"
  },
  "devDependencies": {
    "nodemon": "^1.11.0",
    "standard": "^8.5.0"
  }
}
```

**Comment by thetutlage** [comment]
Duplicate of #357
